tests/utils.go

```diff
@@ -60,6 +66,20 @@ func PodWithPvcSpec(podName, pvcName string, cmd, args []string) *v1.Pod {
 							MountPath: "/pvc",
 						},
 					},
+					SecurityContext: &v1.SecurityContext{
```

Comment:
Just nitpick but most of the time I see people using the pointer library, IMO looks cleaner than having vars for each boolean. No need to change anything though.

Reply:
Yeah I don't disagree